### PR TITLE
404 error page added

### LINKED ISSUE #166: Adding a 404 page
To add a 404 error page to keep a visitor on your site even if he/she hits a page that doesn't exist. Please assign this issue to me @rudrakshi99 as I'm the participant of GSSOC'21.

### PR DESCRIPTION
**Pull request by Ankita297**
# Description
404 error page added
Fixes #166 

## Type of change

- [ x] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ x] This change requires a documentation update

## UI /UX changes
   Attach gif or screenshot for changes.
   - Before :
    - After :
## Checklist:

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] I have made corresponding changes to the documentation
- [x ] My changes generate no new warnings
- [x ] I have added tests that prove my fix is effective or that my feature works
- [x ] New and existing unit tests pass locally with my changes
404 NOT FOUND - Google Chrome 16-05-2021 12_37_49
404 NOT FOUND - Google Chrome 16-05-2021 12_37_57

https://user-images.githubusercontent.com/67833077/118388808-364d5080-b644-11eb-89d0-28de057e0cd2.mp4

### CODE REVIEW

**Comment by Ankita297** [comment]
@harikesh409 @ravikr126 Sir Please review my PR .

File: css/style.css [hunk 1]
```diff
@@ -1018,3 +1018,70 @@ HEAD .round-2 {
     align-items: center;
     margin-left: 7rem;
 }
+
+/* 404 error page   */
+.error-center {
```
Comment (harikesh409) [inline css/style.css:1023]: Can be replaced with bootstrap `text-center` class.

File: 404.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+     <link rel="stylesheet" href="css/style.css">
+    <title>404 NOT FOUND</title>
+</head>
+<body>
+    <div class="error-page">
+    <div class="error-center">
```
Comment (harikesh409) [inline 404.html:12]: Can be replaced with `text-center` class and correct the indentation.

**Review by harikesh409 — CHANGES_REQUESTED** [review]
image

There are 2 scrollbars in certain resolutions and the Go Home button is in the white area instead of the coloured area as shown in your screenshots. Mostly the issue is due to  `.error-page>height:100vh` property.

**Comment by Ankita297** [comment]
done the changes

**Comment by Ankita297** [comment]
@harikesh409 Sir Please review the PR.

**Review by harikesh409 — APPROVED** [review]

**Pull request merged**